packages/assets/scss/01-atoms/_table-responsive.scss

```
@@ -3,24 +3,36 @@
 .ma__table--responsive {
   position: relative;
   margin-bottom: 1em;
-  overflow: hidden;
 
-  &.has-horizontal-scroll {
-    // to accommodate the sticky nav bar
-    padding-bottom: 50px;
+  .ma__table__caption__scroll-info {
+    display: none;
+    font-size: $fonts-medium;
+    font-weight: $fonts-normal;
+
+    &.show {
+      display: inline-block;
+    }
+  }
+
+  #sr-instructions {
+    // Ensure the content is announced via aria-labelledby only.
+    display: none;
   }
 }
 
 .ma__table--responsive__wrapper {
-    overflow-x: scroll;
-    width: 100%;
-    position: relative;
-    -ms-overflow-style: none;  // IE 10+
-    overflow: -moz-scrollbars-none;  // Firefox
 
-    &::-webkit-scrollbar {
-      display: none;  // Safari and Chrome
-    }
+  overflow-x: auto;
+
+  @media ($bp-medium-max) {
+
+    overflow-y: clip;

```

```suggestion
    overflow-y: hidden;
```

No difference than hidden if no [overflow-clip-margin](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-clip-margin) is set. Just want to make sure that this is the desired behavior